### PR TITLE
Mb ai notebooks img seg tuto

### PR DESCRIPTION
**Pull request by MathieuBsqt**
Here is a new documentation that is linked to a new tutorial about image segmentation for AI Notebooks. 

- I have create all language duplicas & updated the index file. 

*This PR will be in draft until the notebook link is available. I need to add it in the docs.*

### CODE REVIEW

File: pages/platform/ai/notebook_tuto_12_image-segmentation-unet-tumors/guide.de-de.md [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+title: AI Notebooks - Tutorial - Brain tumor segmentation using U-Net
+slug: notebooks/tuto-image-segmentation-unet-brain-tumors
+excerpt: How to convert Speech to Text using AI Notebooks
```
Comment (bs-ovh) [inline pages/platform/ai/notebook_tuto_12_image-segmentation-unet-tumors/guide.de-de.md]: Please fix in all files concerned. :)

File: pages/platform/ai/notebook_tuto_12_image-segmentation-unet-tumors/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+>
+> We will train this model on the *[BraTS2020 dataset](http://braintumorsegmentation.org/)*. We will show you how you can easily download the dataset in the notebook tutorial.
+>
+
+## Requirements
+
+- Access to the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB)
+- An AI Notebooks project created inside a [Public Cloud project](https://www.ovhcloud.com/en-gb/public-cloud/) in your OVHcloud account
+- A user for AI Notebooks
+
```
Comment (eleapttn) [inline pages/platform/ai/notebook_tuto_12_image-segmentation-unet-tumors/guide.en-gb.md:35]: I would have probably added:
`- a Kaggle account`

File: pages/platform/ai/notebook_tuto_12_image-segmentation-unet-tumors/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+  	    --gpu <nb-gpus>
+```
+
+You can then reach your notebook’s URL once the notebook is running.
+
+### Accessing the notebooks
+
+Once our [AI examples repository](https://github.com/ovh/ai-training-examples/) has been cloned in your environment, find your notebook by following this path: `ai-training-examples` > `notebooks` > `computer-vision` > `image-segmentation` > `tensorflow` > `brain-tumor-segmentation-unet` > `notebook_image_segmentation_unet.ipynb`.
+
+A preview of this notebook can be found on GitHub [here](####PUT LINK ONCE AVAILABLE####).
```
Comment (eleapttn) [inline pages/platform/ai/notebook_tuto_12_image-segmentation-unet-tumors/guide.en-gb.md]:
```suggestion
A preview of this notebook can be found on GitHub [here](https://github.com/ovh/ai-training-examples/blob/main/notebooks/computer-vision/image-segmentation/tensorflow/brain-tumor-segmentation-unet/notebook_image_segmentation_unet.ipynb).
```

File: pages/platform/ai/notebook_tuto_12_image-segmentation-unet-tumors/guide.en-gb.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+## Go further
```
Comment (eleapttn) [inline pages/platform/ai/notebook_tuto_12_image-segmentation-unet-tumors/guide.en-gb.md:102]:
```suggestion
## Go further

```

**Review by eleapttn — CHANGES_REQUESTED** [review]

**Comment by MathieuBsqt** [comment]
Thanks for your feedbacks ! I have applied all your suggestions.
Now Ready to merge

**Review by eleapttn — APPROVED** [review]

**Review by eleapttn — APPROVED** [review]

**Review by bs-ovh — APPROVED** [review]

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @MathieuBsqt

**Pull request merged**